### ISSUE TITLE
Monsters can walk through walls

### ISSUE THREAD

**Issue by AtillaG1** [Sphereserver/Source-X#1130]
Hi, I have mentioned it in my previous issue, but this might be separate problem.

So we have a tiles (foundations), that have flag "bridge", these are used mainly in static stairs, and allow you to walk stairs into the next level in static houses.
These tiles are also used in many places like dungeons etc....

The problem is, that the monsters can walk through walls that are placed on such foundation with flag "bridge".
![image](https://github.com/Sphereserver/Source-X/assets/25097484/35873a61-e7d4-414d-a0ed-38aad17883bd)
![image](https://github.com/Sphereserver/Source-X/assets/25097484/6c42e9b3-cbe0-4387-bc78-338cb7ee21e1)

https://i.gyazo.com/9dd33ae276b02ba58199500ef74422b0.mp4

**Comment by xwerswoodx** [comment]
Is it a specific location or custom location? If it's exists on map I'd like to get it's coordinates for checks.

**Comment by AtillaG1** [comment]
yes its terathan keep, 5336,1743,5 or 5336,1692,5

**Comment by xwerswoodx** [comment]
May you have different muls on server and client? Because for me at the location there is only normal dungeon for me but I see that stones used at terathan keep and seems like players can pass on it, seems a bit silly but I wonder if it's client-side or sphere-side issue. Or is it issue, because technically they are half blocks and you can walk on half blocks in real life, so they can be done intentionally, but I am not sure, as I see players can walk on water too, it seems a bit stupid honestly. Need to check how sphere works on other half blocks before make a decision.

**Comment by AtillaG1** [comment]
check the video
its in terathan, yes normal dungeon

https://i.gyazo.com/9dd33ae276b02ba58199500ef74422b0.mp4

**Comment by xwerswoodx** [comment]
![image](https://github.com/Sphereserver/Source-X/assets/6155640/f21fa168-8dbe-48c4-8da8-fb8136b965be)

this is where I tp when I enter the coordinates you gave, this is why I asked

**Comment by AtillaG1** [comment]
OH OMG sorry, I didnt realised that i have moved the dungeons to other places.
Its out here.
![image](https://github.com/Sphereserver/Source-X/assets/25097484/8af6f8aa-8402-48fc-8139-02cece9683d4)
![image](https://github.com/Sphereserver/Source-X/assets/25097484/634ff7e0-b5ea-4b6b-b8e1-db4b798dd1c6)

**Comment by xwerswoodx** [comment]
![image](https://github.com/Sphereserver/Source-X/assets/6155640/499ba7f8-4200-435e-8117-2637fb007505)

No no, I am exactly the location you shown on the map, but seems like your map files not comes from original uo (or changed in time), because I am using original muls just update them couple of days ago, if they are custom, there could be some issues, you may need to check the wall flags, because they can be wrong or missing. Even I asked position because it seems buggy for me while the left wall starts middle of the block. 

For the issue, to check I can probably do some tests to see if npcs walk through walls if they are on slabs.

**Comment by xwerswoodx** [comment]
I did some test on my own, but none of them walk through walls for me honestly, I also think walking on that slabs so stupid but other that that, npcs acting normally while moving around.

![image](https://github.com/Sphereserver/Source-X/assets/6155640/a3c95bc1-084a-4b25-a9e0-7528d1e38264)

I removed but there were more than 10 enemies from different types around but none of them goes in, and wrym never leave the area, I don't know what happens on your map but probably can be caused from broken mul files. I can't tell it's on mul-base issue, but honestly everything working fine on my latest build.
